### ISSUE TITLE
ch13 pag 438 no softmax needed

### ISSUE THREAD

**Issue by GianniGi** [rasbt/machine-learning-book#84]
Hello, kindly clarify this: in the nn.Sequential model, there's no softmax at the end, because we use the cross-entropy loss, which probably doesn't need that, because it is equivalent to the combination of LogSoftmax and NLLLoss? Yet the text below says that the output layer is activated by the softmax.

https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html#torch.nn.CrossEntropyLoss Note 

![$Clipboard01](https://user-images.githubusercontent.com/9282332/191790638-a67099e5-aaea-4096-957a-b38e1edd0552.jpg)

**Comment by rasbt** [comment]
Since that's a common misconception, I also wrote a blog post about that. Hah, and I just realize that I never finalized part 2 which was about that specifically! https://sebastianraschka.com/blog/2022/losses-learned-part1.html

**Comment by GianniGi** [comment]
> Yes, you are exactly right. Looking at this paragraph, this looks a bit confusing though and should be explained more properly. Thanks for noting that.

thank YOU :-)

this isn't very clear either, pag 474.

![$$Clipboard01](https://user-images.githubusercontent.com/9282332/191916886-1b0161e3-c168-47b0-878e-b43e571063d6.jpg)

pag 478 instead clears any ambiguity

![$$Clipboard01](https://user-images.githubusercontent.com/9282332/191917062-e993e00a-c04d-4dd9-9c61-2d19bda9e454.jpg)

**Comment by rasbt** [comment]
Thanks for highlighting those! Definitely something that should be made more clear in potential future editions!
